Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -18,12 +18,13 @@ class Installer
 
     PERSISTENT_METADATA_SUBDIRS = ["gpg"].freeze
 
-    def initialize(cask, command: SystemCommand, force: false, skip_cask_deps: false, require_sha: false)
+    def initialize(cask, command: SystemCommand, force: false, skip_cask_deps: false, require_sha: false, reinstall: false)
       @cask = cask
       @command = command
       @force = force
       @skip_cask_deps = skip_cask_deps
       @require_sha = require_sha
+      @reinstall = reinstall
```

Comment:
I think it makes more sense to make `reinstall` a separate method, like `install` and `uninstall`.